tcms/testruns/forms.py

```diff
@@ -26,6 +26,10 @@ class BaseRunForm(forms.Form):
         required=False,
     )
 
+    class Meta:
+        model = TestRun
+        fields = ['notes', 'default_tester', 'build', 'manager', 'summary']
+
     def populate(self, product_id):
         query = {'product_id': product_id, 'is_active': True}
```

Comment:
That is some stupid syntax, leftover from previous iterations. Convert it to kwargs syntax.